LMS.DataAccess/src/FineSystem/Fine.cs

```diff
@@ -0,0 +1,45 @@
+using LMS.DataAccess.BorrowSystem;
+
+namespace LMS.DataAccess.FineSystem;
+
+public class Fine{
+
+    bool Paid;
+    Borrow Borrow;
+
+    int Cost;
+
+    public Fine(Borrow borrow){
+        this.Borrow = borrow;
+        CalculateFine();
+    }
+
+    public void CalculateFine()
+    {
+        DateTime returnDate = DateTime.Now;
+        TimeSpan difference = returnDate - Borrow.GetDueDate();
+
+        int weeksLate = (int)(difference.TotalDays / 7);
+
+        this.Cost = difference.TotalDays > 0 ? 20 + weeksLate * 10 : 0;
+    }
+
+    public void FineDetails()
+    {
+        System.Console.WriteLine("========== Fine Details ==========");
+        System.Console.WriteLine($"Patron Name:          {Borrow.GetPatron().getName()}");
+        System.Console.WriteLine($"Book Title:           {Borrow.GetBook().Title}");
+        System.Console.WriteLine($"Borrow Date:          {Borrow.GetBorrowDate()}");
+        System.Console.WriteLine($"Due Date:             {Borrow.GetDueDate()}");
+        System.Console.WriteLine($"Return Date:          {DateTime.Now}");
+        System.Console.WriteLine($"Fine Amount:          ${Cost}");
+        System.Console.WriteLine($"Paid:                 {(Paid ? "Yes" : "No")}");
+        System.Console.WriteLine("==================================");
+    }
+
+    public bool GetPaid() => Paid;
+
+    public int GetCost() => Cost;
+    public Borrow GetBorrow() => Borrow;
+    public void SetPaid(bool paid) => this.Paid = paid;
+}
```

Comment:
This class also only makes the necessary changes

Reply:
what I mean is that as in all classes you should do only the modification of the namespace and by using